app/package.json

```
@@ -91,7 +91,8 @@
     "cypress:open": "yarn run cypress open",
     "generate:colors": "babel-node --presets \"@babel/env\" scripts/generate-colors.js",
     "release": "babel-node --presets \"@babel/env\" scripts/release.js",
-    "release:finalize": "babel-node --presets \"@babel/env\" scripts/release.js --finalize"
+    "release:finalize": "babel-node --presets \"@babel/env\" scripts/release.js --finalize",
+    "prettier": "prettier \"**/*.@(js|json|svelte)\" --ignore-path .gitignore"
```

```suggestion
    "prettier": "prettier \"**/*.@(js|json|svelte|css|html)\" --ignore-path .gitignore --check"
```
We should also lint the static stuff that we have.

And I think it also makes sense to have the output human readable via `--check`, or did you have a different purpose in mind for this script?

---

Definitely! It's usually useful to add commands to check and write. The first command can stay as a base command to keep configuration and parameters in one place.

We could add:
- `"prettier:check": "yarn prettier --check"
- `"prettier:write": "yarn prettier --write"

Sounds good?

---

Hm, now that I think about it, we actually already have Husky and lint-staged in place which get run just before making a commit and it should bark at us if either the linter or prettier are unhappy.

This is how it looks on my end when I try to commit something funky:
```
$ git commit
husky > pre-commit (node v11.11.0)
  ✔ Preparing...
  ❯ Running tasks...
    ❯ Running tasks for *.{js,css,json,html}
      ✖ prettier --write
    ↓ Running tasks for *.svelte [skipped]
      → No staged files match *.svelte
  ↓ Applying modifications... [skipped]
    → Skipped because of errors from tasks.
  ✔ Reverting to original state...
  ✔ Cleaning up...



✖ prettier found some errors. Please fix them and try committing again.
package.json 37ms

[error] scripts/release.js: SyntaxError: Unexpected token, expected ";" (23:13)
[error]   21 |       console.log("  - https://github.com/github/hub");
[error]   22 |     } else {
[error] > 23 |       raise error;
[error]      |             ^
[error]   24 |     }
[error]   25 |   }
[error]   26 |
husky > pre-commit hook failed (add --no-verify to bypass)
```

I believe prettier is even configured to re-write staged code before the commit. So I wonder how that malformed code even slipped by? Perhaps it was just the missing linter config...

Do you think there is still need for explicit linting and prettifier scripts in this setup?
I'm now leaning towards just mentioning in the README that we have it as a pre-commit hook.